MAUI/DataGrid/column-drag-and-drop.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: post
+title: Column Drag and Drop in .NET MAUI DataGrid control | Syncfusion
+description: Learn all about column drag and drop support in Syncfusion MAUI DataGrid (SfDataGrid) control and more here.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+---
+
+# Column Drag and Drop in MAUI DataGrid (SfDataGrid)
+
+The SfDataGrid allows dragging and dropping a column header by setting the `SfDataGrid.AllowDraggingColumn` property to `true.` The drag view is displayed while dragging a column header. Based on the requirements, drag and drop operations can be handled by using the `SfDataGrid.QueryColumnDragging` event.
```

Comment:
The `SfDataGrid` allows you to drag and drop a column header by setting the `SfDataGrid.AllowDraggingColumn` property to true. A drag view is displayed while dragging a column header. You can handle drag and drop operations based on your requirements by using the `SfDataGrid.QueryColumnDragging` event

Reply:
Modified